testsupport/wait/member.go

```diff
@@ -2351,10 +2365,10 @@ func (a *MemberAwaitility) verifyMutatingWebhookConfig(t *testing.T, ca []byte)
 }
 
 func (a *MemberAwaitility) verifyValidatingWebhookConfig(t *testing.T, ca []byte) {
-	t.Logf("checking ValidatingWebhookConfiguration '%s'", "member-operator-validating-webhook")
+	t.Logf("checking ValidatingWebhookConfiguration '%s'", "member-operator-validating-webhook"+a.Namespace)
 	actualValWbhConf := &admv1.ValidatingWebhookConfiguration{}
-	a.waitForResource(t, "", "member-operator-validating-webhook", actualValWbhConf)
-	assert.Equal(t, bothWebhookLabels, actualValWbhConf.Labels)
+	a.waitForResource(t, "", "member-operator-validating-webhook-"+a.Namespace, actualValWbhConf)
+	assert.True(t, ContainsLabels(actualValWbhConf.Labels, bothWebhookLabels))
```

Comment:
btw, you could have used annotation instead so you could still keep the check for the exact match of the labels

Reply:
good point, I've converted the labels to annotations and reverted those checks in https://github.com/codeready-toolchain/toolchain-e2e/pull/950/commits/9e8562ad953ad0a7571771ca473797e7ee713ce3 

Thanks!